src/ConditionalLogic.php

```diff
@@ -48,31 +49,31 @@ public static function where(string|array $field, string $operator, mixed $value
             throw new InvalidArgumentException("Invalid conditional logic operator [$operator].");
         }
 
-        return new self($field, $operator, $value);
+        return new self($name, $operator, $value, $group);
     }
 
-    public function and(string|array $field, string $operator, mixed $value = null): static
+    public function and(string|array $name, string $operator, mixed $value = null, string|null $group = null): static
     {
-        $this->rules[] = $this->createRule($field, $operator, $value);
+        $this->rules[] = $this->createRule($name, $operator, $value);
         return $this;
     }
 
-    private function createRule(string|array $field, string $operator, mixed $value = null): array
+    private function createRule(string|array $name, string $operator, mixed $value = null, string |null $group = null): array
     {
         return [
-            'field' => is_array($field) ? $field : ['group' => false, 'name' => $field],
+            'name' => $name,
             'operator' => $operator,
             'value' => $value,
+            'group' => $group,
         ];
     }
 
     /** @internal */
     public function get(?string $parentKey = null): array
     {
         return array_map(function ($rule) use ($parentKey) {
-            $rule['name'] = $rule['field']['name'];
-            $parentKey = $rule['field']['group'] ?: $parentKey;
-            unset($rule['field']['group']);
+            $parentKey = $rule['group'] ?: $parentKey;
+            unset($rule['group']);
```

Comment:
This step could probably be skipped since we return a new array, right?

Reply:
I wanted to return only compatible attributes to the final ACF array but it’s optional

Reply:
I agree, in this case the `$rule` variable isn't returned. I guess we could skip the `unset` and save a few 0.00001ms.

Reply:
Fixed in the last commit